locales/en.json

```diff
@@ -1,7 +1,29 @@
 {
```

Comment:
Did I miss the code changes to hook this up to anything?

Reply:
oh, this is all changes from Robert.